From: David Paleino <dapal@debian.org>
Subject: try not to remove upstream files on clean
Forwarded: not-needed

---
 Makefile.am |   23 -----------------------
 1 file changed, 23 deletions(-)

--- a/Makefile.am
+++ b/Makefile.am
@@ -51,27 +51,4 @@ GITIGNOREFILES = build .autotools .cdt*
 
 DISTCHECK_CONFIGURE_FLAGS = --disable-scrollkeeper
 
-CLEANFILES = ChangeLog
-
-# Build ChangeLog from GIT history
-
-CHANGELOG_START = GTHUMB_2_10_11
-ChangeLog:
-	@echo Creating $@
-	@if test -d $(top_srcdir)/.git; then \
-		(GIT_DIR=$(top_srcdir)/.git $(top_srcdir)/missing --run git log $(CHANGELOG_START).. --stat -M -C --name-status  --date=short --no-color) | fmt --split-only > $@.tmp \
-		 && mv -f $@.tmp $@ \
-		 || ($(RM) $@.tmp; \
-		     echo Failed to generate ChangeLog, your ChangeLog may be outdated >&2; \
-		     (test -f $@ || echo git-log is required to generate this file >> $@)); \
-	else \
-		test -f $@ || \
-		(echo A git checkout and git-log is required to generate ChangeLog >&2 && \
-		echo A git checkout and git-log is required to generate this file >> $@); \
-	fi
-
-dist: ChangeLog
-
-.PHONY: ChangeLog
-
 -include $(top_srcdir)/git.mk
