From: David Paleino <dapal@debian.org>
Subject: add missing included header
Origin: vendor
Forwarded: no

---
 extensions/exiv2_tools/exiv2-utils.cpp |    1 +
 1 file changed, 1 insertion(+)

Index: pix-3.2.3/extensions/exiv2_tools/exiv2-utils.cpp
===================================================================
--- pix-3.2.3.orig/extensions/exiv2_tools/exiv2-utils.cpp	2013-08-31 06:28:55.859559782 +1000
+++ pix-3.2.3/extensions/exiv2_tools/exiv2-utils.cpp	2013-08-31 06:28:55.847559723 +1000
@@ -29,6 +29,7 @@
 #include <exiv2/image.hpp>
 #include <exiv2/exif.hpp>
 #include <iostream>
+#include <iomanip>
 #include <string>
 #include <sstream>
 #include <vector>
