Index: gthumb-2.10.7/configure.in
===================================================================
--- gthumb-2.10.7.orig/configure.in	2007-12-28 09:07:19.000000000 +0100
+++ gthumb-2.10.7/configure.in	2007-12-28 09:07:38.000000000 +0100
@@ -5,6 +5,7 @@
 AC_INIT(gthumb, 2.10.7, http://bugzilla.gnome.org/enter_bug.cgi?product=gthumb)
 AC_CONFIG_SRCDIR(src/main.c)
 AM_INIT_AUTOMAKE(AC_PACKAGE_NAME, AC_PACKAGE_VERSION)
+AM_MAINTAINER_MODE
 
 AM_CONFIG_HEADER(config.h)
 
