--- pix.orig/src/Makefile.am
+++ pix/src/Makefile.am
@@ -223,5 +223,7 @@
 	lex.albumtheme.c
 
 CLEANFILES =				\
-	$(gnome_pix_idl_sources)	\
+	X_Pix-stubs.c            \
+	X_Pix-skels.c            \
+	X_Pix-common.c
 	gnome_pix_idl_stamp
--- pix.orig/src/X_Pix.h
+++ pix/src/X_Pix.h
@@ -14,7 +14,6 @@
 
 
 /** typedefs **/
-#line 14 "/usr/share/idl/bonobo-2.0/Bonobo.idl"
 #include <bonobo/Bonobo.h>
 #if !defined(ORBIT_DECL_X_Pix_Application) && !defined(_X_Pix_Application_defined)
 #define ORBIT_DECL_X_Pix_Application 1
